website/package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@keystonejs/website",
   "description": "KeystoneJS documentation. Powered by Gatsby.",
   "private": true,
-  "version": "5.0.4",
+  "version": "5.0.5",
```

Comment:
Our release bot will increment this number for us, based on the contents of the [changeset](https://github.com/atlassian/changesets/blob/master/docs/adding-a-changeset.md). Instead of incrementing this number manually, run `yarn changeset` and select `@keystonejs/website` as the package to release as a `patch`.

Reply:
Reverted in 5ea76ae

Reply:
Changeset added in 0d7c1ac3